examples/vpc_service_control/vpc_sc_project/main.tf

```diff
@@ -0,0 +1,318 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
```

Comment:
provider version should be pinned to 2.6.0

Reply:
I don't know how!  :)

Reply:
I showed how to do it [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/4#discussion_r295053122)